api/tests/test_authentication.py

```diff
@@ -0,0 +1,74 @@
+import string
+from tests import TestsWithMockingDisabled
+
+def strip_punctuation(text):
+    return text.translate(str.maketrans('', '', string.punctuation))
+
+class TestAuthenticationEndpoints(TestsWithMockingDisabled):
```

Comment:
Should we also test a working authentication that returns a 200 status?

If this was left off because a new db instance wont have seeded users, maybe we consider seeding a test user so we can run tests that return a '200' status. what do you think?

Reply:
The tests in my next PR will include valid sign in and sign up attempts. I needed to do some more work to properly isolate test users from real users, to ensure that our test cases did not pollute our AWS data. 

See `https://github.com/hackforla/HomeUniteUs/blob/auth_mocking/api/tests/test_authentication.py` for example tests cases with `2xx` status. Note that the `TestsWithMockingDisabled` is no longer required and all API tests can be run with and without mocking in this upcoming PR.

Reply:
Those test are looking good! Thanks